package.json

```
@@ -14,6 +14,12 @@
     "man/",
     "marked.min.js"
   ],
+  "exports": {
+    ".": {
+      "import": "./lib/marked.esm.mjs",
+      "require": "./src/marked.js"
+    }
```

you should also export `package.json` as some bundlers will want to read it
```suggestion
    },
    "./package.json": "./package.json"
```